src/app/services/backend.service.ts

```diff
@@ -19,84 +19,84 @@ export class BackendService {
     state: string,
     city: string,
     zip_code: number,
-  }={
```

Comment:
Are these properties actually used anywhere? All the data that's eventually sent to the backend should come from components. If these properties aren't being used anywhere, it's best to delete them.

Reply:
Just pushed up the resolve.